MaxMind.GeoIP2/WebServiceClient.cs

```diff
@@ -15,6 +15,10 @@
 using System.Text;
 using System.Threading.Tasks;
 
+#if NETSTANDARD2_0 || NETSTANDARD2_1

```

Comment:
For this and below `#if !NET45 && !NETSTANDARD1_4` might be better given that all future targets should also include this. As is, someone might add a target without updating the list.

Reply:
Changed.

Reply:
Fixed.